deploy/backplane/hybridsre-hcp/hypershift/management-cluster/20-hybridsre-hcp.SubjectPermission.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-hybridsre-hcp
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - clusterRoleName: hypershift-readers
```

Comment:
does `dedicated-readers` function as a superset of both these groups?

Reply:
Indeed it does. I should just switch the whole thing to dedicated-readers then?

Reply:
Switched to dedicated-readers.